doc/releasenotes/15_0_0_release_notes.md

```diff
@@ -1,47 +1,47 @@
-- **[Online DDL Changes](#a-idonline-ddl-changesonline-ddl-changes)**
-  - [Concurrent Vitess Migrations](#a-idconcurrent-vitess-migrationsconcurrent-vitess-migrations)
-  - [VTCtl Command Changes](#a-idvtctl-command-changesvtctl-command-changes)
-  - [New Syntax](#a-idnew-syntaxnew-syntax)
-- **[Tablet Throttler](#a-idtablet-throttlertablet-throttler)**
-  - [API Changes](#a-idapi-changesapi-changes)
-- **[Mysql Compatibility](#a-idmysql-compatibilitymysql-compatibility)**
-  - [System Settings](#a-idsystem-settingssystem-settings)
-  - [Lookup Vindexes](#a-idlookup-vindexeslookup-vindexes)
-- **[Durability Policy](#a-iddurability-policydurability-policy)**
-  - [Cross Cell](#a-idcross-cellcross-cell)
-- **[New EXPLAIN Format](#a-idnew-explain-formatnew-explain-format)**
-  - [FORMAT=vtexplain](#a-idformatvtexplainformatvtexplain)
-- **[VTOrc](#a-idvtorcvtorc)**
-  - [Old UI Removal and Replacement](#a-idold-ui-removal-and-replacementold-ui-removal-and-replacement)
-  - [Configuration Refactor and New Flags](#a-idconfiguratoin-refactor-and-new-flagsconfiguration-refactor-and-new-flags)
-  - [Example Upgrade](#a-idexample-upgradeexample-upgrade)
-  - [Default Configuration Files](#a-iddefault-configuration-filesdefault-configuration-files)
-- **[Flags Restructure](#a-idflags-restructureflags-restructure)**
-  - [Flags Diff](#a-idflag-diffflags-diffd)
+- **[Breaking Changes](#breaking-changes)**
+  - [Flags](#flags)
+  - [VTTablet Flag Deletions](#vttablet-flag-deletions)
+  - [Vindex Interface](#vindex-interface)
+- **[Deprecations](#deprecations)**
+  - [LogStats Table and Keyspace Deprecated](#logstats-table-and-keyspace-deprecated)
+  - [Orchestrator Integration Deprecation](#orchestrator-integration-deprecation)
+  - [Connection Pool Prefill](#connection-pool-prefill)
+  - [InitShardPrimary Deprecation](#initshardprimary-deprecation)
+- **[Command-Line Syntax Deprecations](#command-line-syntax-deprecations)**
+  - [VTTablet Startup Flag Deletions](#vttablet-startup-flag-deletions)
+  - [VTTablet Startup Flag Deprecations](#vttablet-startup-flag-deprecations)
+  - [VTBackup Flag Deprecations](#vtbackup-flag-deprecations)
+- **[VTGate](#vtgate)**
+  - [vtgate --mysql-server-pool-conn-read-buffers](#vtgate--mysql-server-pool-conn-read-buffers)
+- **[VDiff2](#vdiff2)**
+  - [Resume Workflow](#resume-workflow)
+  - [vtctl GetSchema --table-schema-only](#vtctl-getschema--table-schema-only)
+  - [Support for Additional Compressors and Decompressors During Backup & Restore](#support-for-additional-compressors-and-decompressors-during-backup--restore)
+  - [Independent OLAP and OLTP Transactional Timeouts](#independant-olap-and-oltp-transactional-timeouts)
+  - [Support for Specifying Group Information in Calls to VTGate](#support-for-specifying-group-information-in-calls-to-vtgate)
```

Comment:
These should be moved to their own sections or be placed under a `Misc` / `Other` section. They are not related to vdiff2.